openmetadata-ui/src/main/resources/ui/src/hooks/useApplicationStore.ts

```diff
@@ -153,4 +156,38 @@ export const useApplicationStore = create<ApplicationStore>()((set, get) => ({
   setAppVersion: (version: string) => {
     set({ appVersion: version });
   },
+  refetchCurrentUser: async (params?: {
```

Comment:
Lets not add api fetching logic inside store. Store should only contain getters and setters.